packages/popover/src/Popover.tsx

```diff
@@ -219,7 +222,7 @@ function Popover({
               display: none;
             `}
           />
-          <Root>
+          <Root {...rootProps}>
```

Comment:
How come we aren't inlining the className here?
```js
<Root className={usePortal ? portalClassName : ''}>
```

The intent here seems a bit more clear than the `rootProps` constant above.

Reply:
Since `Root` can be a `React.Fragment`, always passing in a `className` prop would give a runtime warning:
```Warning: Invalid prop `className` supplied to `React.Fragment`. React.Fragment can only have `key` and `children` props.```